### ISSUE TITLE
Guidance on entries in bibliography / references section

### ISSUE THREAD

**Issue by heidivanparys** [w3c/wcag#2193]
### Question

Would it be possible to add guidance and examples for writing a bibliography / references section that conforms to the Web Content Accessibility Guidelines?

A typical entry could look like the following:

[WCAG21] W3C. Web Content Accessibility Guidelines (WCAG) 2.1. W3C Recommendation. 5 June 2018. Available from: https://www.w3.org/TR/WCAG21/

How should such an entry be adapted in order to conform with the guidelines?

### Background

When using this style of writing references for a document that I worked on, I got the feedback that the URL should be available on the title, together with a reference to the good practices described at https://www.w3.org/WAI/tips/writing/#make-link-text-meaningful.

As I would like to keep the style of the bibliography unchanged, at least in the printed version, I created a temporarily work-around by following [H33: Supplementing link text with the title attribute](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H33), so the reference would like this:

```xml
<p>[WCAG21] W3C. Web Content Accessibility Guidelines (WCAG) 2.1. W3C Recommendation. 5 June 2018. Available from: <a href="https://www.w3.org/TR/WCAG21/" title="Web Content Accessibility Guidelines (WCAG) 2.1.">https://www.w3.org/TR/WCAG21/</a></p>
```

resulting in:

<p>[WCAG21] W3C. Web Content Accessibility Guidelines (WCAG) 2.1. W3C Recommendation. 5 June 2018. Available from: <a href="https://www.w3.org/TR/WCAG21/" title="Web Content Accessibility Guidelines (WCAG) 2.1.">https://www.w3.org/TR/WCAG21/</a></p>

I then got the idea to look at [the references in the aforementioned W3C Recommendation](https://www.w3.org/TR/WCAG21/#references), where I can see that

- all references include the URL in the text itself (using a slightly different style: "URL: ..." instead of "Available from:")
- a link has been added on the title

```xml
<p>[WCAG21] W3C. <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines (WCAG) 2.1</a>. W3C Recommendation. 5 June 2018. Available from: <a href="https://www.w3.org/TR/WCAG21/">https://www.w3.org/TR/WCAG21/</a></p>
```

resulting in:

<p>[WCAG21] W3C. <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines (WCAG) 2.1</a>. W3C Recommendation. 5 June 2018. Available from: <a href="https://www.w3.org/TR/WCAG21/">https://www.w3.org/TR/WCAG21/</a></p>

Note: running https://www.w3.org/TR/WCAG21/ through the [WAVE Accessibility Extension](https://addons.mozilla.org/en-GB/firefox/addon/wave-accessibility-tool/) gives alerts on “Redundant links”:

![image](https://user-images.githubusercontent.com/11427611/150505469-66a34e4c-2162-4f34-adb2-9892e59cae8b.png)

**Comment by fstrr** [comment]
This issue is labelled as a discussion, so we’re moving this to Discussions. There doesn’t seem to be an update to make to the documentation, but if that changes, we can move it back to the issues list.
